randovania/generator/hint_distributor.py

```diff
@@ -335,13 +346,18 @@ async def assign_precision_to_hints(
 
         if hint_kind == HintNodeKind.GENERIC:
             enable = player_pool.configuration.hints.enable_random_hints
+            is_specific = False
+
         elif hint_kind == HintNodeKind.SPECIFIC_LOCATION:
             enable = player_pool.configuration.hints.enable_specific_location_hints
+            is_specific = True
+
         elif hint_kind == HintNodeKind.SPECIFIC_PICKUP:
```

Comment:
not that familiar with the hint part of the codebase. How can hint SPECIFIC_PICKUP hint kinds be made now? Since you moved the one to SPECIFIC_LOCATION

Reply:
they can't, and never could be

Reply:
not really relevant to this PR